cwms-data-api/src/main/java/cwms/cda/api/CountyController.java

```diff
@@ -58,6 +59,13 @@
 public class CountyController implements CrudHandler {
     private final MetricRegistry metrics;
     private final Histogram requestResultSize;
+    private static final ContentTypeAliasMap CONTENT_TYPE_ALIAS_MAP = new ContentTypeAliasMap();
+
+    static
+    {
```

Comment:
This kinda makes sense, the but it requires extra dev time configuration.
There's an annotation for the supported formatters that could have "default = true" added to it and the Formats processor can pick it up. 

Then we can set the default format at the DTO level, which is were we also establish the allow formats.
(Granting we have to do it again anyways at the moment due to how OpenAPI works.)

Reply:
Oh, annotations are a much better idea for this, especially at the DTO level.  I didn't notice that part of the design, thank you for bringing this up Mike.